.gitmodules

```diff
@@ -39,3 +39,7 @@
 	path = repos/xgcm/xgcm-examples
 	url = https://github.com/xgcm/xgcm-examples.git
 	branch = binderbot-built
+[submodule "repos/pangeo-gallery/ncar-gallery"]
+	path = repos/pangeo-gallery/ncar-gallery
+	url = https://github.com/NCAR/notebook-gallery.git
+	branch = binderbot-built
```

Comment:
Yes, I'm referring to this part. If you try to visit it, you can see it doesn't exist: https://github.com/NCAR/notebook-gallery/tree/binderbot-built